### PR TITLE
fix(ci): markdownlint ci not work

### PR DESCRIPTION
**Pull request by jiangfucheng**
### Description

Fixes https://github.com/apache/apisix/issues/9242
### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/getting-started.md [hunk 1]
```diff
@@ -143,13 +143,13 @@ curl -sL https://run.api7.ai/apisix/quickstart | sh
 curl "http://127.0.0.1:9080" --head | grep Server
 ```
 
-I如果一切顺利，将输出如下信息。
+I 如果一切顺利，将输出如下信息。
```
Comment (monkeyDluffy6017) [inline docs/zh/latest/getting-started.md:146]: The `I` is redundant

**Comment by monkeyDluffy6017** [comment]
Could you show me the ci link of  `doc-lint ci seems not work`? Are you sure this pr can fix this?

**Comment by jiangfucheng** [comment]
Sorry, it was my mistake. I tested it in my own forked repository and it worked well, so I mistakenly thought that the issue was caused by `”“`. However, the doc-lint CI is really doesn't work anymore as evidenced by the numerous failed actions in https://github.com/apache/apisix/actions/workflows/doc-lint.yml. 

I have researched some sources but I am unsure if they are related to this issue. Please refer to: https://github.com/orgs/community/discussions/25488

Regarding this PR, if you believe that these document modifications are useful and should be merged, I will remove any unnecessary code and keep it clean. Otherwise, I will close it.

**Comment by soulbird** [comment]
> Sorry, it was my mistake. I tested it in my own forked repository and it worked well, so I mistakenly thought that the issue was caused by `”“`. However, the doc-lint CI is really doesn't work anymore as evidenced by the numerous failed actions in https://github.com/apache/apisix/actions/workflows/doc-lint.yml.
> 
> I have researched some sources but I am unsure if they are related to this issue. Please refer to: https://github.com/orgs/community/discussions/25488
> 
> Regarding this PR, if you believe that these document modifications are useful and should be merged, I will remove any unnecessary code and keep it clean. Otherwise, I will close it.

So， just remove any unnecessary code and keep it clean

**Comment by monkeyDluffy6017** [comment]
@jiangfucheng Some of the changes seem unnecessary, perhaps you should raise a separate PR to fix these documentation issues, with a new PR starting with "docs: "

**Comment by jiangfucheng** [comment]
@soulbird @monkeyDluffy6017 Thank you for your reply, I will create new PR to fix these documentation issues in my spare time.

**Comment by jiangfucheng** [comment]
@soulbird @monkeyDluffy6017  Thank you for your reply, I will create new PR to fix these documentations issues in my spare time.

**Pull request closed without merging**